src/image_builder.py

```diff
@@ -174,18 +175,18 @@ def image_builder(buildspec):
     # In the context of the ThreadPoolExecutor each instance of image.build submitted
     # to it is executed concurrently in a separate thread.
     with concurrent.futures.ThreadPoolExecutor(max_workers=10) as executor:
-        # Standard images must be built before example images
-        # Example images will use standard images as base
-        standard_images = [image for image in IMAGES if "example" not in image.name.lower()]
-        example_images = [image for image in IMAGES if "example" in image.name.lower()]
+        # Parent images do not inherit from any containers built in this job
```

Comment:
nice. This will bring a lot of clarity.

Reply:
The changes should be synced up with #1186 as both are targeted to revamp the build system